src/vulkan/pipeline.h

```diff
@@ -48,6 +49,10 @@ class Pipeline {
 
   Result AddDescriptor(const BufferCommand*);
 
+  // Add information of how and what to do with push constant to
```

Comment:
// Add a new set of values in an offset range to the push constants to be used on the next pipeline execution.

Reply:
done